media_driver/linux/common/codec/ddi/media_ddi_encode_hevc.cpp

```diff
@@ -115,7 +115,7 @@ VAStatus DdiEncodeHevc::ContextInitialize(
     codecHalSettings->pCpParams = m_encodeCtx->pCpDdiInterface->GetParams();
 
     VAStatus eStatus = VA_STATUS_SUCCESS;
-
+    m_encodeCtx->targetUsage = TARGETUSAGE_RT_SPEED;
```

Comment:
Can the HEVC target usage also be initialized to BEST_QUALITY? Or the default usage of AVC/MPEG2 is set to RT_SPEED?

Reply:
yes, all will be unified set default  to RT_SPEED